zjit/src/hir.rs

```diff
@@ -4698,6 +4709,7 @@ mod tests {
         bb0(v0:BasicObject, v1:BasicObject):
           v3:BasicObject = GetLocal l0, EP@3
           v5:BasicObject = Send v3, 0x1000, :each
+          v6:BasicObject = GetLocal l0, EP@3
```

Comment:
In this case, if there's anything/what would access `v6`? Or is the `GetLocal` just for `state.setlocal`?